faiss/gpu/test/TestGpuIndexFlat.cpp

```diff
@@ -73,6 +73,9 @@ void testFlat(const TestFlatOptions& opt) {
     faiss::gpu::GpuIndexFlatConfig config;
     config.device = device;
     config.useFloat16 = opt.useFloat16;
+#if defined USE_NVIDIA_RAFT
```

Comment:
shouldn't we test both with and without RAFT?

Reply:
Yes we should. We've just turned this on in the meantime so we can get all the tests passing from the RAFT side.